src/pages/younghyunKim/Login/Login.js

```diff
@@ -1,8 +1,93 @@
 import React, { Component } from "react";
+import "./Login.scss";
 
 class LoginYoungHyun extends Component {
+  state = {
+    id: "",
+    pw: "",
+    isActive: false,
+  };
+
+  handleInput = e => {
+    const { className, value } = e.target;
+    this.setState({
+      [className]: value,
+      isActive:
+        this.state.id.indexOf("@") !== -1 && this.state.pw.length > 5
+          ? true
+          : false,
+    });
+  };
```

Comment:
```suggestion
  handleInput = e => {
    const { name, value } = e.target;    
    const isActive = id.indexOf("@") !== -1 && pw.length > 5
    
    this.setState({ [name]: value, isActive });
  };
```
- className보다 name 속성 써주는게 더 좋아보이네요. className은 스타일링에 사용할 수도 있으니까요
- 구분되는 로직은 한줄 띄워주세요
- 구조분해 할당도 적용해주시고
- 불리언 값을 그대로 활용할 수 있는 경우에는 삼항연산자 불필요합니다

Reply:
감사합니다. 불 필요한 삼항연사자 삭제하고 진행하였습니다!